build.gradle.kts

```diff
@@ -43,6 +46,10 @@ repositories {
 }
 
 val licenseHeaderFile = file("gradle/license-header.txt")
+fun readCopyrightHeader(licenseHeaderFile: File) =
+    licenseHeaderFile.readLines()
+        .joinToString("\n") { line -> line.substring(min(line.length, 3)) }
```

Comment:
Probably not in scope, but could this be simplified with one of the many `trimIndent` and friends variants?

Reply:
Merging this PR, I think this could be in scope for "simplify build.gradle"